src/components/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
each react component should be moved to a folder with its name, folder should include the component file and file with component styles